docs/UserGuide.md

```diff
@@ -196,68 +202,136 @@ Format: `clear`
+**:information_source: Note:** 
+
+This example is based of [`Adding a tag`](#adding-a-tag-to-a-contact-tag-add). Feel free to replace `Test` and `Test2` with any existing tag and a fresh new tag.
 </div>
 
-* Adds a `TAG` to the contact at the specified `INDEX`.
+2. Press `Enter` on your keyboard.
+3. You should now see `Tag has changed from [Test] to [Test2]` in the Command result box. Images of Before and After are provided below. Within each contact, those which have `Test` tag, will now display `Test2` instead.
 
-Example:
-* `tag add 1 t/friend` adds the friend tag to the first contact shown in the list.
+before:
+(insert image a contact with `Test` tag)
+
+After:
+(insert image of the same contact with `Test2` tag instead of `Test`)
+
+4. Fantastic! You have successfully changed the `Test` tag to the `Test2` tag.
 
 ### Removing a Tag from a Contact: `tag remove`
 
-Removes an existing tag from an existing contact.
+You can remove a tag from a contact.
 * `Coming soon in v1.5`, we will upgrade `tag remove` to remove tags from todos.
 
 Format: `tag remove INDEX t/TAG`
 
-* Removes a `TAG` from the contact at the specified `INDEX`.
+Steps to take:
+1. Input `tag remove 1 t/Test2` into the Command box. 
 
-Example:
-* `tag remove 1 t/friend` removes the friend tag from the first contact shown in the list.
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:** 
+
+This example is based of [`Editing a Tag`](#editing-a-tag-tag-edit). Feel free to replace `1` and `Test2` with any contact index and an existing tag within the contact indexed.
+</div>
+
+2. Press `Enter` on your keyboard.
+3. You should now see `Tag removed: [Test2]` in the Command result box just like the image below.
+
+(insert image of contact `1` without the `Test2` tag)
+
+4. Nice! You have successfully removed a tag from a contact.
+
+### Deleting a Tag: `tag delete`
+
+You can delete a tag from the tag list.
+
+Format: `tag delete t/TAG`
+
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:** When `TAG` is deleted, `TAG` is removed from all the contacts which previously had it.
+</div>
+
+Steps to take:
+1. Input `tag delete t/Test2` in the Command box. 
+
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:** 
+
+This example is a follow-up of [`removing a Tag`](#removing-a-tag-from-a-contact-tag-remove). Feel free to replace `Test2` wit any existing tag.
+</div>
+
+2. Press `Enter` on your keyboard.
+3. You should now see `Tag deleted: [Test2]` in the Command result box just like the image below.
+
+(insert successful deletion og `Test2` tag)
+
+4. Terrific! You have successfully deleted a tag.
+
+### Viewing tags `[Coming soon in v1.5]`
+
+Now that you made it here, you have successfully mastered the tag features! Refer to the [Commannd summary](#command-summary) for a quick reference to all tag commands. Let us move on to the [Customisation Features](#customisation-features).
```

Comment:
```suggestion
Congratulations on completing this section on Tag Management Features, you have successfully mastered the tag features! Refer to the [Command summary](#command-summary) for a quick reference to all tag commands. Let us move on to the [Customisation Features](#customisation-features).
```

Reply:
I don't know how I feel about this section of text. Well it is a great idea, the current position of it makes it appear as the description for the `Viewing Tags` feature. On the other hand, I have no good ideas on where to position it or the header title it would have